UPGRADE-3.x.md

```diff
@@ -1,6 +1,17 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.5 to 3.6
```

Comment:
@greg0ire shall this headline be removed or not necessary?

Reply:
I think it should stay